specification/search/data-plane/Azure.Search/preview/2024-05-01-Preview/searchindex.json

```diff
@@ -0,0 +1,3097 @@
+          "description": "The threshold will filter based on the '@search.score' value. Note this is the @search.score returned as part of the search response. The threshold direction will be chosen for higher @search.score."
+        }
+      },
+      "required": [
+        "value"
+      ],
+      "description": "The results of the vector query will filter based on the '@search.score' value. Note this is the @search.score returned as part of the search response. The threshold direction will be chosen for higher @search.score."
+    },
+    "VectorThresholdKind": {
+      "type": "string",
+      "enum": [
+        "vectorSimilarity",
+        "searchScore"
+      ],
+      "x-ms-enum": {
+        "name": "VectorThresholdKind",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "vectorSimilarity",
+            "name": "VectorSimilarity",
+            "description": "The results of the vector query will be filtered based on the vector similarity metric. Note this is the canonical definition of similarity metric, not the 'distance' version. The threshold direction (larger or smaller) will be chosen automatically according to the metric used by the field."
+          },
+          {
+            "value": "searchScore",
+            "name": "SearchScore",
+            "description": "The results of the vector query will filter based on the '@search.score' value. Note this is the @search.score returned as part of the search response. The threshold direction will be chosen for higher @search.score."
+          }
+        ]
+      },
+      "description": "The kind of vector query being performed."
+    },
+    "VectorFilterMode": {
+      "type": "string",
+      "enum": [
+        "postFilter",
+        "preFilter"
+      ],
+      "x-ms-enum": {
+        "name": "VectorFilterMode",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "postFilter",
+            "name": "PostFilter",
+            "description": "The filter will be applied after the candidate set of vector results is returned. Depending on the filter selectivity, this can result in fewer results than requested by the parameter 'k'."
+          },
+          {
+            "value": "preFilter",
+            "name": "PreFilter",
+            "description": "The filter will be applied before the search query."
+          }
+        ]
+      },
+      "description": "Determines whether or not filters are applied before or after the vector search is performed."
+    },
+    "HybridSearch": {
+      "type": "object",
+      "properties": {
+        "maxTextRecallSize": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Determines the maximum number of documents to be retrieved by the text query portion of a hybrid search request. The values needs to be between 1 and 100 000. Default is 1000. A higher value allows paging through more documents, at the cost of a higher latency."
```

Comment:
Mike - explain more clearly how this is different than top

Reply:
My recollection from the API review: this value determines the number of documents to factor into the document weighting, which can affect the document scores. It has nothing to do with how many documents are returned.

Reply:
Thank you! Was just going off the transcript

Reply:
I'll try to reformulate this. It does indeed affect how many documents are returned, since it defines the maximum amount of documents that we will consider for the text query. Example, if you set the Value of MaxTextRecallSize to 100, and the K value for the vector query to 10, then the maximum amount of documents that can be returned is 110 (100 + 10, assuming there's no overlap between the text results and the vector results).